### DISCUSSION TITLE
For the coder

### DISCUSSION THREAD

**Discussion by Stoppedwumm** [Stoppedwumm/StoppedwummDownloader#17] [Announcements]
# We need more
## Why
We have an issue, specifically #14, I tried fixing, and I don't know if I did it right.

## What do I have to do
I'm going to compile in a few seconds. Please test and debug.

## Final Notes
Maybe I fucked something up, if that's the case, you are allowed to spam Issues and pinging me. Then I will revert the changes in #16.

### Have a great day

**Comment by StoppedwummSites** [comment]
We are done, please do not respond anymore, the test is now closed
